packages/documentation/.storybook/styles/preview.scss

```diff
@@ -128,6 +128,13 @@ $monospace: SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier
     }
   }
 
+  [data-color-scheme='dark'] {
+    code {
```

Comment:
I'm not sure this is useful as the color-scheme is set on the storybook preview elements, and not the text content of the stories.

Reply:
This is useful, if you use `code` elements in your snapshot stories, because the `schemes` wrapper function uses also the `data-color-scheme` attribute.